### PR TITLE
added routes in Dessau

### PR DESCRIPTION
**Pull request by lunas-bad-coding**

### CODE REVIEW

File: line-colors.csv [hunk 1]
```diff
@@ -94,6 +94,25 @@ db-regionetz-westfrankenbahn,RE 80,db-regionetz-verkehrs-gmbh-westfrankenbahn,re
...
+dvg-dessau,17,,5-nas001-16,#FFCC00,#ffffff,,pill
+dvg-dessau,21,,5-nas001-16,#EE722E,#ffffff,,pill
+dvg-dessau,22,,5-nas001-16,#0F3F93,#ffffff,,pill
+dvg-dessau,N1,,5-nas001-n1,#000000,#ffffff,,pill
+dvg-dessau,N2,,5-nas001-n2,#000000,#ffffff,,pill
+dvg-dessau,N3,,5-nas001-n3,#000000,#ffffff,,pill
+dvg-dessau,N4,,5-nas001-n4,#000000,#ffffff,,pill
+dvg-dessau,N5,,5-nas001-n5,#000000,#ffffff,,pill
+dvg-dessau,N6,,5-nas001-n6,#000000,#ffffff,,pill
+dwe-dessau,DWE,,dwe,#ffffff,#000000,#000000,rectangle
```
Comment (vainamov) [inline line-colors.csv:115]: Hey, thank you for your contribution! Can you change the hex values to use lowercase letters?

```suggestion
dvg-dessau,1,,8-nasdvg-1,#f59c00,#ffffff,,rectangle
dvg-dessau,3,,8-nasdvg-3,#f59c00,#ffffff,,rectangle
dvg-dessau,10,,5-nas001-10,#2e76bb,#ffffff,,pill
dvg-dessau,11,,5-nas001-11,#95d2f1,#ffffff,,pill
dvg-dessau,12,,5-nas001-12,#529bb9,#ffffff,,pill
dvg-dessau,13,,5-nas001-13,#9ec753,#ffffff,,pill
dvg-dessau,14,,5-nas001-14,#e3000b,#ffffff,,pill
dvg-dessau,15,,5-nas001-15,#a80126,#ffffff,,pill
dvg-dessau,16,,5-nas001-16,#941680,#ffffff,,pill
dvg-dessau,17,,5-nas001-16,#ffcc00,#ffffff,,pill
dvg-dessau,21,,5-nas001-16,#ee722e,#ffffff,,pill
dvg-dessau,22,,5-nas001-16,#0f3f93,#ffffff,,pill
dvg-dessau,N1,,5-nas001-n1,#000000,#ffffff,,pill
dvg-dessau,N2,,5-nas001-n2,#000000,#ffffff,,pill
dvg-dessau,N3,,5-nas001-n3,#000000,#ffffff,,pill
dvg-dessau,N4,,5-nas001-n4,#000000,#ffffff,,pill
dvg-dessau,N5,,5-nas001-n5,#000000,#ffffff,,pill
dvg-dessau,N6,,5-nas001-n6,#000000,#ffffff,,pill
dwe-dessau,DWE,,dwe,#ffffff,#000000,#000000,rectangle
```

Comment (lunas-bad-coding) [reply line-colors.csv:115]: of course!

File: line-colors.csv [hunk 2]
```diff
@@ -94,6 +94,25 @@ db-regionetz-westfrankenbahn,RE 80,db-regionetz-verkehrs-gmbh-westfrankenbahn,re
...
+dvg-dessau,17,,5-nas001-16,#ffcc00,#ffffff,,pill
+dvg-dessau,21,,5-nas001-16,#ee722e,#ffffff,,pill
+dvg-dessau,22,,5-nas001-16,#0f3f93,#ffffff,,pill
+dvg-dessau,N1,,5-nas001-n1,#000000,#ffffff,,pill
+dvg-dessau,N2,,5-nas001-n2,#000000,#ffffff,,pill
+dvg-dessau,N3,,5-nas001-n3,#000000,#ffffff,,pill
+dvg-dessau,N4,,5-nas001-n4,#000000,#ffffff,,pill
+dvg-dessau,N5,,5-nas001-n5,#000000,#ffffff,,pill
+dvg-dessau,N6,,5-nas001-n6,#000000,#ffffff,,pill
+dwe-dessau,DWE,,dwe,#ffffff,#000000,#000000,rectangle
```
Comment (marhei) [inline line-colors.csv:115]: This operator is missing in the sources.json file!

**Review by marhei — CHANGES_REQUESTED** [review]

File: line-colors.csv (hunk 2 above)
Comment (lunas-bad-coding) [reply line-colors.csv:115]: my bad, sorry!

**Review by marhei — APPROVED** [review]

**Pull request merged**